cli/src/commands/get/sources.rs

```diff
@@ -35,6 +44,26 @@ pub fn get(client: &Client, args: &GetSourcesArgs, printer: &Printer) -> Result<
         .map(|bucket| (bucket.id.clone(), bucket))
         .collect();
 
+    let source_stats: HashMap<_, _> = if *include_stats {
+        sources
+            .iter()
+            .map(|source| {
+                info!("Getting statistics for source {}", source.full_name().0);
+                let stats = client
+                    .get_source_statistics(
+                        &source.full_name(),
+                        &StatisticsRequestParams {
+                            comment_filter: Default::default(),
+                        },
+                    )
+                    .expect("Could not get statistics for source");
+                (source.id.clone(), stats)
+            })
+            .collect()
```

Comment:
Should raise error rather than `expect`ing here, as per previous reviews:
```suggestion
        sources
            .iter()
            .map(|source| {
                info!("Getting statistics for source {}", source.full_name().0);
                let stats = client
                    .get_source_statistics(
                        &source.full_name(),
                        &StatisticsRequestParams {
                            comment_filter: Default::default(),
                        },
                    )
                    .context("Could not get statistics for source")?;
                Ok((source.id.clone(), stats))
            })
            .collect()?
```
or something like this